storage/src/index/mod.rs

```diff
@@ -0,0 +1,19 @@
+//! A memory-efficient index for values stored in a journal.
```

Comment:
nit: I don't think we need to imply this can only be used `in a journal`

Reply:
dropped "in a journal"